.skillsaw/plugindocs_rule.py

```
@@ -0,0 +1,90 @@
+            return violations
+
+        data_json_path = context.root_path / "docs" / "data.json"
+
+        website_script_path = context.root_path / "scripts" / "build-website.py"
+        if not website_script_path.exists():
+            return violations
+
+        try:
+            original_data_json = data_json_path.read_text() if data_json_path.exists() else None
+
+            result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)